examples/docile_documentation/build.jl

```diff
@@ -0,0 +1,82 @@
+import Lexicon.Elements: document, section, page, docs, config
+
+import Lexicon.Utilities: iscategory, isexported
+
+import Lexicon.Render: markdown, save
+
+
+import Docile
+import Docile: Cache, Collector, Extensions, Formats, Interface, Legacy, Runner, Utilities
+
+cd(dirname(@__FILE__)) do
+
+    pages  = [page(
+                "# $m",
+                "## External",
+                "### Modules, Functions, Methods & Macros",
+                docs(m, filter = obj -> iscategory(m, obj, [:module, :function, :method, :macro]) && isexported(m, obj)),
```

Comment:
Maybe you have an idea how to do this.

I would like to only write:

`iscategory(obj, [:module, :function, :method, :macro]), isexported(obj)),` withouth the additional module reference.

The code it calls is [here](https://github.com/MichaelHatherly/Lexicon.jl/pull/116/files#diff-01285fe4a25bbc4da684aa18148e934eR72)

```julia
function innerpage!(io::IO, n::Node, m::Module, emptyline::Bool)
    warn("!! TODO: `Page Docs Module` not finished yet!!")
    objects = Cache.objects(m)
    # get any filter config for this node
    conf = findconfig(n)
    objs = haskey(conf, :filter) ? filter(conf[:filter], objects) : objects

    for obj in objs
    .....
```

You also had in mind to change `conf = findconfig(n)` to return the value for `conf = findconfig(n, :filter)`